requirements.txt

```diff
@@ -62,3 +62,4 @@ Authlib==0.15.5
 advocate==1.0.0
 urllib3==1.26.16
 importlib-resources==5.13.0
+duckdb==0.8.0
```

Comment:
i think you need move row to https://github.com/getredash/redash/blob/master/requirements_all_ds.txt

Reply:
yes. Done